docs/src/howto-guides/028_c8y_service_monitoring.md

```diff
@@ -0,0 +1,60 @@
+# How to monitor health of service from Cumulocity IoT
+
+The health of a `thin-edge.io` service or any other `service` that is running on the `thin-edge.io` device
+or on the `child` device can be monitored from the Cumulocity by sending the `health-status` message to `Cumulocity IoT`.
```

Comment:
General change (in multiple places), we should used the markdown `**Cumulocity IoT**` when referring to Cumulocity (e.g. in bold, and always using a capital letter for Cumulocity)

```suggestion
or on the `child` device can be monitored from the **Cumulocity IoT** by sending the `health-status` message to **Cumulocity IoT**.
```